biogeochem/EDPhysiologyMod.F90

```diff
@@ -679,17 +747,22 @@ subroutine phenology_leafonoff(currentSite)
              !DROUGHT LEAF OFF
              if (currentSite%dstatus == 1)then        
                 if (currentCohort%status_coh == 2)then ! leaves have not dropped
+
+                   ! This sets the cohort to the "leaves off" flag
                    currentCohort%status_coh      = 1   
-                   currentCohort%laimemory   = currentCohort%bl
-                   ! add retranslocated carbon (very small) to store.      
-                   currentCohort%bstore      = currentCohort%bstore        
-                   ! add falling leaves to litter pools . convert to KgC/m2                    
-                   currentCohort%leaf_litter = currentCohort%bl  
-                   currentCohort%bl          = 0.0_r8                                        
+                   
+                   ! Remember what the lai (leaf mass actually) was for next year
+                   currentCohort%laimemory   = leaf_c
```

Comment:
Huh?  "next" year?  Also why do we consider leaf mass also and area?  Personally I think that is confusing

Reply:
I guess technically it should be leafc_memory... Do we still need this, @rgknox, in the presence of the new flushing parameter and the target-centric allocation scheme. It might be better, although not bfb, to have leaf flushing as a fraction of the target and forget about lai_memory?

Reply:
I am fine with that @rosiealice .  It seems a little cleaner to me, but I didn't want to mess with existing stuff.  If it has your blessing I will remove laimemory and just use the target.